docs/configuration/miscellaneous.md

```diff
@@ -121,6 +121,15 @@ Default: False
 
 Setting this to True will display a "maintenance mode" banner at the top of every page. Additionally, NetBox will no longer update a user's "last active" time upon login. This is to allow new logins when the database is in a read-only state. Recording of login times will resume when maintenance mode is disabled.
 
+## BANNER_MAINTENANCE
```

Comment:
This should go between `BANNER_LOGIN` and `BANNER_TOP` as all config parameters are ordered alphabetically.